### PR TITLE
did some code style fixes, nothing changed function wise

### PR DESCRIPTION
**Pull request by oded-regev**

### CODE REVIEW

File: ConcentrationGame/ConcentrationModel.swift [hunk 1]
```diff
@@ -23,47 +23,47 @@ class ConcentrationModel
...
-                    cards[cardIndex].isMatch=true
-                    cards[matchIndex].isMatch=true
+                    cards[cardIndex].isMatch = true
+                    cards[matchIndex].isMatch = true
                     score+=2
                 }
                 else
                 {
-                    if cards[cardIndex].countFlips>0
+                    if cards[cardIndex].countFlips > 0
```
Comment (nogahcp) [inline ConcentrationGame/ConcentrationModel.swift:41]: Why did you make this change?

File: ConcentrationGame/ViewController.swift [hunk 2]
```diff
@@ -46,11 +46,11 @@ class ViewController: UIViewController {
     }
         
     @IBAction func NewGame(_ sender: UIButton) {
-        game=ConcentrationModel(numberOfCardsPairs: (cards.count+1)/2)
-        theme=themes.randomElement() ?? Theme(cardColor: #colorLiteral(red: 1, green: 0.5763723254, blue: 0, alpha: 1), backgroundColor: #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1), cardEmojis: ["🎃","🙀","🦇","😈","🏴‍☠️","☠️","👻","🧟‍♂️"])
-        cardsEmoji = Dictionary<Int,String>()
-        emojis=theme.cardEmojis
-        updateViewFromModel()
+        self.game = ConcentrationModel(numberOfCardsPairs: (cards.count+1)/2)
```
Comment (nogahcp) [inline ConcentrationGame/ViewController.swift:49]: did you tested this function?

**Review by nogahcp — COMMENTED** [review]
nice work, see my comments inside

File: ConcentrationGame/ConcentrationModel.swift (hunk 1 above)
Comment (oded-regev) [reply ConcentrationGame/ConcentrationModel.swift:41]: Because I want to improve the code

File: ConcentrationGame/ViewController.swift (hunk 2 above)
Comment (oded-regev) [reply ConcentrationGame/ViewController.swift:49]: yes, it works :)

**Review by nogahcp — APPROVED** [review]

**Pull request merged**